contracts/WorkspaceRegistry.sol

```diff
@@ -19,8 +19,8 @@ contract WorkspaceRegistry is Ownable, Pausable, IWorkspaceRegistry {
     /// @notice mapping to store workspaceId vs workspace data structure
     mapping(uint96 => Workspace) public workspaces;
 
-    /// @notice mapping to store workspaceId vs admins
-    mapping(uint96 => mapping(address => bool)) public workspaceAdmins;
+    /// @notice mapping to store workspaceId vs members vs roles
+    mapping(uint96 => mapping(address => bytes32)) public getUserRoles;
```

Comment:
Why do you need 32 bytes for this information?

Reply:
Poor variable name.
GetUserRole points at being a function that needs to be called
Verbs for methods, nouns for variables.